process-engine/bpmn-studio
# ✨ Update Waiting Room

## What did you change?

This PR adjusts the waiting room to the management api.
Also the Notifications for not supported form field types has been improved.

## How can others test the changes?

- Open up the BPMN-Studio
- Run any diagram with user tasks
- **Enjoy the waiting room**
- Wait for the user task
- Confirm the user task
- **Enjoy the waiting room**
- Notice that one gets redirected after the process has been finished

## PR-Checklist

Please check the boxes in this list after submitting your PR:
